operatorconfig/moduleconfig/authorization/v1.11.0/container.yaml

```diff
@@ -1,6 +1,6 @@
 name: karavi-authorization-proxy
 imagePullPolicy: IfNotPresent
-image: dellemc/csm-authorization-sidecar:v1.8.0
+image: dellemc/csm-authorization-sidecar:v1.11.0
```

Comment:
isn't last release is 1.10.0?

Reply:
![backtothefuture](https://github.com/dell/csm-operator/assets/13646596/5e48a191-b3f8-4984-88af-89ff71bfaa5d)